frontend/src/views/WheresMyGene/common/types.ts

```diff
@@ -1,10 +1,13 @@
 import { CSSProperties } from "react";
 
-export interface CellTypeAndGenes {
```

Comment:
Hey @seve ! Madison and I went through the type names again to make sure it conveys the meanings as efficiently as possible 😆 Can you please take another look at the new names and see if they look good to you too?

Thanks a lot!